packages/next-swc/crates/next-core/js/src/overlay/internal/helpers/magic-identifier.ts

```diff
@@ -18,90 +18,91 @@ const enum Mode {
   LongHex,
 }
 
-const DECODE_REGEX = /^__TURBOPACK__([a-zA-Z0-9_$]+)__$/;
+const DECODE_REGEX = /^__TURBOPACK__([a-zA-Z0-9_$]+)__$/
 
 function decodeMagicIdentifier(identifier: string): string {
-  const matches = identifier.match(DECODE_REGEX);
+  const matches = identifier.match(DECODE_REGEX)
   if (!matches) {
-    return identifier;
+    return identifier
   }
 
-  const inner = matches[1];
+  const inner = matches[1]
 
-  let output = "";
+  let output = ''
 
-  let mode: Mode = Mode.Text;
-  let buffer = "";
+  let mode: Mode = Mode.Text
+  let buffer = ''
   for (let i = 0; i < inner.length; i++) {
-    const char = inner[i];
+    const char = inner[i]
 
     if (mode === Mode.Text) {
-      if (char == "_") {
-        mode = Mode.Underscore;
-      } else if (char == "$") {
-        mode = Mode.Hex;
+      if (char == '_') {
```

Comment:
These could be constants.

```suggestion
      if (char == CHAR_MODE_UNDERSCORE) {
```